src/cartservice/HealthImpl.cs

```diff
@@ -14,11 +15,14 @@ internal class HealthImpl : HealthBase {
         }
 
         public override Task<HealthCheckResponse> Check(HealthCheckRequest request, ServerCallContext context){
-            Console.WriteLine ("Checking CartService Health");
-
-            return Task.FromResult(new HealthCheckResponse {
+            var watch = StartNew();
+            var result = Task.FromResult(new HealthCheckResponse {
                 Status = dependency.Ping() ? HealthCheckResponse.Types.ServingStatus.Serving : HealthCheckResponse.Types.ServingStatus.NotServing
             });
+            watch.Stop();
+            var elapsedMs = watch.ElapsedMilliseconds;
+            Console.WriteLine ("✅ Health Check took " + elapsedMs + "ms");
```

Comment:
You might want to remove this if you added it just for debugging. I am not sure if it will generate a lot of logs.